index.js

```diff
@@ -161,4 +161,6 @@ is.inRange = (x, range) => {
 	throw new TypeError(`Invalid range: ${util.inspect(range)}`);
 };
 
+is.domElement = x => is.object(x) && x.nodeType === 1;
```

Comment:
I think this is a bit too loose of a check.

Not sure what else we should check, but see https://www.google.co.th/search?q=is+dom+element+javascript+check&oq=is+dom+element+javascript+check&aqs=chrome..69i57j0l2j69i64l3.4818j0j1&sourceid=chrome&ie=UTF-8 and https://github.com/lodash/lodash/blob/master/isElement.js for some ideas.

Reply:
Yes, I do agree now. Any object with a nodeType would be selected as a domElement. I will change this behavior and update the pull request.